.github/workflows/deploy-to-balena.yml

```diff
@@ -1,38 +0,0 @@
-name: "Deploy to Balena"
-
-on:
-  pull_request:
-    types: [opened, synchronize, closed]
-    branches:
-      - main
-      - master
-    paths-ignore:
-      - 'docs/**'
-      - README.md
-
-  workflow_dispatch:
-
-env:
-  BALENA_APP: balenalabs/balenasound
-
-jobs:
-  build:
-    name: "Deploy-to-Balena"
-    runs-on: "ubuntu-latest"
-    steps:
-      - name: Checkout main branch
-        uses: actions/checkout@v3
-
-      - name: Update balena.yml
-        uses: balena-io-experimental/balena.yml-action@main
-        with:
-          # Synchronise the GitHub README with the Balena Hub README
-          sync_readme: true
-          # If pushing a tag to deploy, use the same tag as the version number to display on Balena Hub
-          sync_tag: false
```

Comment:
@vipulgupta2048 @alanb128 i think you might want to keep this. IIRC, it updates the `balena.yml` file so that the `README` is up to date on balenaHub

Reply:
I thought BalenaHub started to pickup the GitHub readme retroactively without extra steps?

Reply:
i wish 😓

Reply:
Can readd this then? Let me know if I should?